Firmware/menu.cpp

```diff
@@ -242,25 +226,19 @@ static void menu_draw_item_puts_E(char type_char, const Sheet &sheet)
     lcd_set_cursor(0, menu_row);
     SheetFormatBuffer buffer;
     menu_format_sheet_E(sheet, buffer);
-    lcd_printf_P(PSTR("%c%-18.18s%c"), menu_selection_mark(), buffer.c, type_char);
+    lcd_putc(menu_selection_mark());
+    lcd_print_pad_P(buffer.c, LCD_WIDTH - 2);
+    lcd_putc(type_char);
 }
 
 static void menu_draw_item_puts_P(char type_char, const char* str, char num)
 {
     lcd_set_cursor(0, menu_row);
-    lcd_printf_P(PSTR("%c%-.16S "), menu_selection_mark(), str);
+    lcd_putc(menu_selection_mark());
+    lcd_print_pad_P(str, LCD_WIDTH - 3);
```

Comment:
This usage is not correct. This uses only the length limiting modifier, not also the padding modifier.

Reply:
You are correct 👍 Interestingly it's not an issue with "Extruder X" items.
![IMG_2832](https://user-images.githubusercontent.com/8218499/212473386-9acf2307-b232-493f-82d3-3df990201081.jpg)

![IMG_2831](https://user-images.githubusercontent.com/8218499/212473397-8ce5e122-3caf-4214-9aa3-4929e43d37e4.jpg)

Reply:
Issue is now fixed. I changed `lcd_print_pad_P` such that it now returns the number of padded pads (the remains of `len`) It can help us determine the next cursor position.